### ISSUE TITLE
CloudComPy for Mac?

### ISSUE THREAD

**Issue by kxd195** [CloudCompare/CloudComPy#104]
Just wondering if there's any plan to support Mac or is the Linux build modifiable to run on Mac?

**Comment by prascle** [comment]
Yes, there is a plan to support Mac! I'll start working on it in 2 or 3 weeks, and I hope to provide a first version for June...

Regards,
Paul

**Comment by prascle** [comment]
A first version of CloudComPy is available for tests, and CloudCompare is also provided. See the [readme](https://github.com/CloudCompare/CloudComPy/blob/master/doc/UseMacOSCondaBinary.md).

Regards,
Paul
